docs/understand/data-structure.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Data Structure
+parent: Understanding lakeFS
+description: Understand the data structure in lakeFS
+---
+
+# How Does lakeFS Store Your Data
+lakeFS being a data versioning engine, requires the ability to save multiple versions of the same object. As a result, lakeFS stores objects in the object store
+in an unconventional way.
+This might cause confusion when trying to understand where our data is actually being stored. This page will try to shed a light on this subject.
+
+## lakeFS Repository Namespace Structure
+lakeFS stores repository data and metadata under the repository's namespace.
+Listing a repository storage namespace in the object store will provide the following output:
+
+```shell
+aws s3 ls s3://<storage_namespace>/
+                        PRE _lakefs/
+                        PRE data/
+```
+
+lakeFS stores the actual user data under the `data/` prefix. The `_lakefs/` prefix is used to store commit metadata which includes [range and meta-range](../understand/how/versioning-internals.md) files and internal lakeFS data.
+Since lakeFS manages immutable data, objects are not saved using their logical name. This means that when you upload a csv file called allstar_games_stats.csv to branch main, lakeFS will generate a random physical
+address under the `data/` prefix and upload it to there.  
+Mapping from a path to an object changes as you upload, commit, and merge on lakeFS. When updating an object, lakeFS will create a new physical address for that version preserving the different version of that object.
+lakeFS will link between the objects logical address and physical address and store that relation under the given commit metadata (range and meta-range)
```

Comment:
```suggestion
lakeFS will link between the object's logical address and its physical address - and store that relation under the given commit metadata (range and meta-range)
```

Reply:
Already did that in line 22. Do you want me to add the link also here?